src/electionguard/auxiliary.py

```diff
@@ -41,3 +42,33 @@ class AuxiliaryPublicKey(NamedTuple):
 
 AuxiliaryDecrypt = Callable[[ENCRYPTED_MESSAGE, SECRET_KEY], Optional[MESSAGE]]
 """A callable type that represents the auxiliary decryption scheme."""
+
+
+def identity_auxiliary_encrypt(message: str, public_key: str) -> Optional[str]:
```

Comment:
i dont think we should include this in the core lib

Reply:
Ok. I'll manually make lambdas in the functions.